### ISSUE TITLE
Use `tilde.json` to locate users

### ISSUE THREAD

**Issue by Tweska** [Tweska/TildeverseGallery#5]
We can replace the user parsers for ~club and ~town if we use the `tilde.json` file instead.

- https://tilde.club/tilde.json
- http://tilde.town/tilde.json

**Comment by shesha4572** [comment]
Hey!! Could you check the pr out?

**Comment by Tweska** [comment]
Yeah, I had a peek and it looks good but I didn't have the time to properly test and merge it yet.

**Comment by shesha4572** [comment]
cool!
